### DISCUSSION TITLE
Power feed type color

### DISCUSSION THREAD

**Discussion by krasotinpa** [netbox-community/netbox#13765] [General]
Hi all

Is there a way to change color of Power feed type field?
I would like to change color of 'redundant' choice.

Thank in advance.

**Comment by atom-tr** [comment]
`dcim.PowerFeed.type` do not supported to replace with setting in `configuration.py`. 
Check out: https://docs.netbox.dev/en/stable/configuration/data-validation/

Not recommended, but u can change it inside: `netbox/dcim/choices.py`

**Comment by krasotinpa** [comment]
Thanks atom-tr.
I've made appropriate changes to `netbox/dcim/choices.py` on our Netbox installation.
Now `configuration.py` accepts config like this:

```
FIELD_CHOICES = {
    'dcim.PowerFeed.type': (
        ('primary', 'Primary', 'blue'),
        ('redundant','Redundant', 'red'),
    ),
}

```
Submitted a new feature request for it.

**Comment by dejantep** [comment]
Quick question...
Where exactly is meant to add "Field_choices"?

just below "Custom Validators" in configuration.py or just about anywhere?

**Reply by candlerb** [reply]
Anywhere in configuration.py is fine.  [Doc](https://docs.netbox.dev/en/stable/configuration/data-validation/#field_choices).  Note that it's `FIELD_CHOICES` in caps.
